src/Momento.Sdk/Internal/ScsDataClient.cs

```diff
@@ -16,16 +14,16 @@ namespace Momento.Sdk.Internal;
 public class ScsDataClientBase : IDisposable
 {
     internal readonly DataGrpcManager grpcManager;
-    private readonly uint defaultTtlSeconds;
-    private readonly uint dataClientOperationTimeoutMilliseconds;
-    private readonly ILogger _logger;
+    protected readonly int defaultTtlMilliseconds;
+    protected readonly int dataClientOperationTimeoutMilliseconds;
+    protected readonly ILogger _logger;
     protected readonly CacheExceptionMapper _exceptionMapper;
 
-    public ScsDataClientBase(IConfiguration config, string authToken, string endpoint, uint defaultTtlSeconds)
+    public ScsDataClientBase(IConfiguration config, string authToken, string endpoint, TimeSpan defaultTtl)
     {
         this.grpcManager = new(config, authToken, endpoint);
-        this.defaultTtlSeconds = defaultTtlSeconds;
-        this.dataClientOperationTimeoutMilliseconds = config.TransportStrategy.GrpcConfig.DeadlineMilliseconds;
+        this.defaultTtlMilliseconds = (int)defaultTtl.TotalMilliseconds;
```

Comment:
this cast here jumps out at me.  What is the actual type of that value?  And should we be considering changing the type of our variable to match it rather than casting?

Reply:
Good catch. The deadline is a `DateTime`, so we do not need to convert to milliseconds (unlike before). I've changed this in  4528491. There is no longer any casting in deadline path.